auto_round/mllm/autoround_mllm.py

```diff
@@ -170,10 +166,15 @@ def __init__(
                 logger.warning(f"{model.config.model_type} not support for {dataset},"
                             " will use liuhaotian/llava_conv_58k with default config as an alternative.")
             dataset = "liuhaotian/llava_conv_58k"
-            self.truncation = False
+
+        if dataset not in CALIB_DATASETS.keys():
+            truncation = False
             batch_size = 1
-            gradient_accumulate_steps = 8
-            seqlen = 512
+            seqlen = min(512, seqlen) if seqlen is not None else 512
```

Comment:
if seqlen is None,  logger.waring and reset it to 512.  otherwise, just use the seqlen, no min op is needed

Reply:
sorry, reset it to 2048